sdk/lib/ucrt/string/i386/_memicmp.s

```diff
@@ -54,77 +56,90 @@ page
 ;Exceptions:
 ;
 ;*******************************************************************************
+#endif
 
-        CODESEG
+        .code
```

Comment:
Maybe you can restore the original `CODESEG`, now that you've added a define for it in asm.inc.